res/values-pt-rBR/strings.xml

```diff
@@ -816,7 +816,7 @@
     <string name="Next_Tournament">Próximo Torneio</string>
     <string name="Plasma_Won">Plasma Ganhado</string>
     <string name="Semi_Finals">Semi-Finais</string>
-    <string name="Quarter_Finals">Quartas de Finais</string>
+    <string name="Quarter_Finals">Quartas de Final</string>
```

Comment:
Please keep it as "Finais", because the previous text was already based on english text and, consequently, was already correct.